index.html

```diff
@@ -9,6 +9,32 @@
   <link rel="stylesheet" href="//cdn.jsdelivr.net/npm/docsify@4.10.2/lib/themes/vue.css">
   <script src="https://cdn.jsdelivr.net/npm/katex-copytex@1.0.2/dist/katex-copytex.min.js"></script>
   <link href="https://cdn.jsdelivr.net/npm/katex-copytex@1.0.2/dist/katex-copytex.min.css" rel="stylesheet" type="text/css">
+  <style>
+
+.meeting_registration {
+	display: flex;
+	width: 100%;
+	height: 100%;
+	flex-direction: column;
+	overflow: hidden;
+}  
+
+div.meeting_registration > iframe {
+	height:1400px;
```

Comment:
I've cut it at a lower height intentionally to not show the footer.

```suggestion
	height:910px;
```

Reply:
This doesn't work when the iframe switches to a single column layout though :(

Reply:
What's "this"? Your new height or something else? (and using what browser)

When it's the height, the general problem is that you can't determine the height of the page included in the iframe with CSS only. So in the end you experimentally have to find the right value (or live with an unfavorably cropped page).

Reply:
Sorry, I indeed meant the height. I am aware that the iframe contents are not accessible.

The `910` value I indeed determined by trial and error, but it only works for two column layout (which is probably more common using screens).